pkg/cmd/server/admin/default_certs.go

```diff
@@ -82,6 +82,18 @@ func DefaultAPIClientCerts(certDir string) []ClientCertInfo {
 		DefaultClusterAdminClientCertInfo(certDir),
 		DefaultRouterClientCertInfo(certDir),
 		DefaultRegistryClientCertInfo(certDir),
+		DefaultProxyClientCertInfo(certDir),
+	}
+}
+
+func DefaultProxyClientCertInfo(certDir string) ClientCertInfo {
```

Comment:
I've liked the way that groups have worked out for us in the past.  Are you against a group in principle or just couldn't think of a reason for it in this case?